packages/cli/src/lib/browser.js

```diff
@@ -1,5 +1,14 @@
+/*
+* Rendertron - Modified
+* Repo: https://github.com/Goo./rendererndertron
```

Comment:
should fix this with the correct repo https://github.com/GoogleChrome/rendertron

Reply:
oh right!  good catch